src/main/java/seedu/duke/Duke.java

```diff
@@ -10,40 +10,41 @@
 
 public class Duke {
 
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static final String HELLO_MESSAGE = "Hello from\n" + LOGO + "What is your name?";
+    private static final String HELP_PROMPT_MESSAGE = "Enter 'help' for the list of available commands";
+
     /**
      * Main entry-point for the java.duke.Duke application.
      */
     public static void main(String[] args) throws Exception {
-
         Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
         logger.setLevel(Level.WARNING);
 
         Shelf warehouse = new Shelf("warehouse");
 
-
-        String logo = " ____        _        \n"
-            + "|  _ \\ _   _| | _____ \n"
-            + "| | | | | | | |/ / _ \\\n"
-            + "| |_| | |_| |   <  __/\n"
-            + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("What is your name?");
+        System.out.println(HELLO_MESSAGE);
```

Comment:
No need this, there is a static method to print welcome message in the UI package